examples/compiled/area_density.vg.json

```diff
@@ -15,7 +15,8 @@
           "type": "kde",
           "field": "IMDB Rating",
           "bandwidth": 0.3,
-          "as": ["value", "density"]
+          "as": ["value", "density"],
+          "resolve": "shared"
```

Comment:
This used to be `independent`. https://vega.github.io/vega/docs/transforms/kde/

Why should this be shared for non-stacked transforms? Maybe a reasonable default is to have it only shared if there is stacking?

Reply:
If I remember correctly from our discussion, we said it would be difficult to set the default value of the transform based on whether the encoding channel is stacked or not. I believe the main reason was that it would be hard to detect which encoding channel the density values are plotted on.

Leaving "shared" as the default would work in most situations, including most of the common ones, and if there are situations where the "independent" is needed, it would be easy to switch. To avoid that densities stack by default, we said we would turn off stacking as part of a `mark_density`, but leave the default stacking behavior for the area mark as it is now to be consistent with other marks.